pdctl/command/namespace_command.go

```diff
@@ -86,11 +97,31 @@ func appendTableCommandFunc(cmd *cobra.Command, args []string) {
 	}
 	if _, err := strconv.Atoi(args[1]); err != nil {
 		fmt.Println("table_id shoud be a number")
+		return
+	}
+
+	input := make(map[string]interface{})
+	input["namespace"] = args[0]
+	input["table_id"] = args[1]
+	input["action"] = "append"
+
+	postJSON(cmd, namespaceRelPrefix, input)
+}
+
+func removeTableCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 2 {
+		fmt.Println("Usage: namespace remove <name> <table_id>")
+		return
+	}
+	if _, err := strconv.Atoi(args[1]); err != nil {
+		fmt.Println("table_id shoud be a number")
+		return
 	}
 
 	input := make(map[string]interface{})
 	input["namespace"] = args[0]
 	input["table_id"] = args[1]
+	input["action"] = "remove"
 
-	postJSON(cmd, namespaceAddPrefix, input)
+	postJSON(cmd, namespaceRelPrefix, input)
```

Comment:
Why use another URL as you have defined different action fields?

Reply:
The old URL may confuse user as it also support 'delete' function.